WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -535,7 +535,7 @@ def _submitJob(self, jobID, jobParams, resourceParams, optimizerParams,
 
     wrapperFile = result['Value']
     jobReport.setJobStatus(status='Matched',
-                           minor='Could Not Extract JDL Parameters',
+                           minor='Submitted To CE',
```

Comment:
The wrapper is not yet submitted at this point

```suggestion
                           minor='Submitting To CE',
```

Reply:
Done.